ChangeLog.md

```diff
@@ -1,3 +1,9 @@
+# 1.12.1.0
+
+Andreas Abel, 2023-04-02
+* Add `sPrintCommandsFn` which can be specified to override
```

Comment:
`sPrintCommandsFn` is just an internal field, not visible to the users of the `shelly` library, if I am not mistaken.
The changelog should list changes relevant to the users, so changes of the exported API.  This would be `echoWith` and `print_commands_with`, AFAICT.

Reply:
Oh right, fixed